src/main/java/mate/academy/springbootproject/model/Book.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.springbootproject.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.util.Objects;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted = false")
+@Entity
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "title", nullable = false)
+    private String title;
+    @Column(name = "author", nullable = false)
+    private String author;
+    @Column(name = "isbn", nullable = false, unique = true)
+    private String isbn;
+    @Column(name = "price", nullable = false)
+    private BigDecimal price;
+    @Column(name = "description", length = 1000)
+    private String description;
+    @Column(name = "cover_image")
+    private String coverImage;
+    @Column(name = "is_deleted", nullable = false)
+    private boolean isDeleted = false;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Book book = (Book) o;
+        return Objects.equals(id, book.id)
+                && Objects.equals(title, book.title)
+                && Objects.equals(author, book.author)
+                && Objects.equals(isbn, book.isbn)
+                && Objects.equals(price, book.price)
+                && Objects.equals(description, book.description)
+                && Objects.equals(coverImage, book.coverImage);
```

Comment:
Think do we need to compare all fields? We have ISBN as a unique identifier. Maybe we can` return Objects.equals(isbn, book.isbn);` ?